experiment-descriptor.json

```diff
@@ -35,12 +35,33 @@
       "content-type": "text"
     },
     {
-      "target": "simulation.html",
+      "unit-type": "task",
+      "label": "Pretest",
+      "content-type": "assesment",
+      "source": "pretest.json",
+      "target": "pretest.html"
+    },
+    {
+      "target": "demo.html",
+      "source": "simulation/simulation.html",
```

Comment:
Can we name these files based on the functions they have, i.e. demo.html and practice.html?

Reply:
Yes we can. I have updated the file.